app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="container">
+  <div id="expenses" class="expense-container">
+    <%= render @expenses %>
+  </div>
+</div>
+
+<div class="bg-secondary fixed-bottom d-flex justify-content-center py-3">
+  <%= link_to new_group_expense_path, class: "w-75 d-flex align-items-center justify-content-center gap-2 py-2 px-5 bg-secondary border text-white block font-medium" do %>
+    <span>New transaction</span>
+  <% end %>
```

Comment:
Kindly maintain a consistent design that follows these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) for Colors and Typography. 

- The `new Transaction` button should be the same as your `Add new category` button.

Your `New Transaction` button:

![image](https://user-images.githubusercontent.com/38088818/221356390-c50e4b81-c781-4409-8afe-89eb5790676b.png)

Your `Add new Category` button:

![image](https://user-images.githubusercontent.com/38088818/221356563-2dc27fe3-c71e-4eaf-96e5-d27ca356538c.png)

Reply:
Hi @KDlamini , Just fixed the button for the transactions too.